src/binary-reader-objdump.cc

```diff
@@ -414,26 +411,27 @@ static Result on_import(uint32_t index,
                         StringSlice module_name,
                         StringSlice field_name,
                         void* user_data) {
```

Comment:
We could, yes. No strong opinion either way, but I left it in because there might be unknown new imports one day (which we couldn't deal with without code changes anyway, so...remove?)